coderd/database/dbmem/dbmem.go

```diff
@@ -389,6 +389,12 @@ func (q *FakeQuerier) convertToWorkspaceRowsNoLock(ctx context.Context, workspac
 
 		rows = append(rows, wr)
 	}
+	if withSummary {
+		rows = append(rows, database.GetWorkspacesRow{
+			Name:  "**TECHNICAL_ROW**",
+			Count: count,
+		})
+	}
```

Comment:
I'm not 100% sold on the technical summary row. While it's not strictly compatibility-breaking, it's something that older clients won't know how to handle properly.

That having been said, given that it's an opt-in flag, it's not as big a deal.